demos/browser/app/meetingV2/video/VideoTileCollection.ts

```diff
@@ -246,6 +245,10 @@ export default class VideoTileCollection implements AudioVideoObserver {
     this.updateLayout();
   }
 
+  private isLocalAttendee(attendeeId: string) {
+    return attendeeId.startsWith(this.localAttendeeId);
```

Comment:
Why use `startsWith`, not `equal`?

Reply:
This checks content as well, I can add a comment